internal/pkg/cli/job_deploy.go

```diff
@@ -342,6 +342,8 @@ func (o *deployJobOpts) runtimeConfig(addonsURL string) (*stack.RuntimeConfig, e
 		AddonsTemplateURL:        addonsURL,
 		AdditionalTags:           tags.Merge(o.targetApp.Tags, o.resourceTags),
 		ServiceDiscoveryEndpoint: endpoint,
+		AccountID:                o.targetApp.AccountID,
```

Comment:
Can we add this code to `job_package.go` and `svc_package.go`?

Reply:
Added and manually checked job deploy, job package, svc deploy, and svc package.

Reply:
Relevant CF generated from svc package:

```yaml
  TaskDefinition:
    Metadata:
      'aws:copilot:description': 'An ECS task definition to group your containers and run them on ECS'
    Type: AWS::ECS::TaskDefinition
    DependsOn: LogGroup
    Properties:
      Family: !Join ['', [!Ref AppName, '-', !Ref EnvName, '-', !Ref WorkloadName]]
      NetworkMode: awsvpc
      RequiresCompatibilities:
        - FARGATE
      Cpu: !Ref TaskCPU
      Memory: !Ref TaskMemory
      ExecutionRoleArn: !Ref ExecutionRole
      TaskRoleArn: !Ref TaskRole
      ContainerDefinitions:
        - Name: !Ref WorkloadName
          Image: !Ref ContainerImage
          
          # We pipe certain environment variables directly into the task definition.
          # This lets customers have access to, for example, their LB endpoint - which they'd
          # have no way of otherwise determining.
          Environment:
          - Name: COPILOT_APPLICATION_NAME
            Value: !Sub '${AppName}'
          - Name: COPILOT_SERVICE_DISCOVERY_ENDPOINT
            Value: test.pubapp.local
          - Name: COPILOT_ENVIRONMENT_NAME
            Value: !Sub '${EnvName}'
          - Name: COPILOT_SERVICE_NAME
            Value: !Sub '${WorkloadName}'
          - Name: COPILOT_SNS_TOPIC_ARNS
            Value: '{"publisher1":"arn:aws:sns:us-west-2:xxxxxxxxxx:pubapp-test-testpubsvc-publisher1"}'

...

#from taskrole policy addition
      Policies:
        - PolicyName: 'DenyIAMExceptTaggedRoles'
          PolicyDocument:
            Version: '2012-10-17'
            Statement:
              - Effect: 'Deny'
                Action: 'iam:*'
                Resource: '*'
              - Effect: 'Allow'
                Action: 'sts:AssumeRole'
                Resource:
                  - !Sub 'arn:${AWS::Partition}:iam::${AWS::AccountId}:role/*'
                Condition:
                  StringEquals:
                    'iam:ResourceTag/copilot-application': !Sub '${AppName}'
                    'iam:ResourceTag/copilot-environment': !Sub '${EnvName}'
        - PolicyName: 'ExecuteCommand'
          PolicyDocument:
            Version: '2012-10-17'
            Statement:
              - Effect: 'Allow'
                Action: [
                  "ssmmessages:CreateControlChannel",
                  "ssmmessages:OpenControlChannel",
                  "ssmmessages:CreateDataChannel",
                  "ssmmessages:OpenDataChannel"
                ]
                Resource: "*"
              - Effect: 'Allow'
                Action: [
                  "logs:CreateLogStream",
                  "logs:DescribeLogGroups",
                  "logs:DescribeLogStreams",
                  "logs:PutLogEvents"
                ]
                Resource: "*"
  
  ...
  
  publisher1SNSTopic:
      Type: AWS::SNS::Topic
      Properties:
        TopicName: !Sub '${AWS::StackName}-publisher1'
  publisher1SNSTopicPolicy:
      Type: AWS::SNS::TopicPolicy
      DependsOn: publisher1SNSTopic
      Properties:
        Topics: 
          - !Ref publisher1SNSTopic
        PolicyDocument:
          Version: '2012-10-17'
          Statement:
            - Effect: Allow
              Principal:
                Service: sqs.amazonaws.com
              Action: 
                - sns:Subscribe
              Resource: !Sub 'arn:${AWS::Partition}:sqs:${AWS::Region}:${AWS::AccountId}:*'
              Condition:
                StringEquals:
                  'sqs:ResourceTag/copilot-application': !Sub '${AppName}'
                  'sqs:ResourceTag/copilot-environment': !Sub '${EnvName}'
                ForAnyValue:StringEquals:
                  'sqs:ResourceTag/copilot-service':
                    - 'worker1'
```